### DISCUSSION TITLE
Feature request: Copy "git checkout"-compliant string from PR

### DISCUSSION THREAD

**Discussion by gomesalexandre** [community/community#11481] [Pull Requests]
Currently, when a contributor has a fork and you want to check it out locally, you have to:
- add remote
- git checkout remoteName/branchName

On the PR page, it is displayed that:
`<contributor name> wants to merge x commit/s into <targetBranch> from <contributorUsername>:<contributorSourceBranch>)` with a way to copy the source path.

Unfortunately, this format isn't compatible with git checkout:

```
❯ git checkout <contributorUsername>:<contributorSourceBranch>
error: pathspec <contributorUsername>:<contributorSourceBranch> did not match any file(s) known to git
```

**We then have to change the `:` char for a `/` in the terminal to check out the branch.**
It would be a great value to be able to either:
- Have a button to copy the `git checkout` compliant string OR
- Have a button to add the remote and checkout in one go

Alternatively, there could be two separate buttons, one to easily checkout `git remote add <contributorUsername> <remoteUrl>`,  the second to copy the checkout string, both directly from the PR page since this is common actions in the reviewing flow.

**Comment by xmo-odoo** [comment]
The issue also exists from the other side: if I'm the contributor and the maintainers have made some remarks I need to fix locally, the copy button gives me `<login>:<branch>` requiring manipulation as the only bit I want is `<branch>` in order to locally check it out and update it.
